test/index.test.js

```diff
@@ -1117,4 +1117,225 @@ describe('index', () => {
             });
         });
     });
+
+    describe('_addWebhook', () => {
+        const oauthToken = 'oauthToken';
+        const scmUri = 'bitbucket.org:arnoldWeber/wyattLib:mybranch';
```

Comment:
But the format is correct. Someone who has no context on this might look at this test and think scmUri is host:owner/reponame:branch as opposed to host:owner:uuid/branch. I think we should keep it as close to the correct format as possibile to avoid confusion